src/core/nixl_p2p_metadata_backend.cpp

```
@@ -0,0 +1,392 @@
+    }
+    return true;
+}
+
+bool
+recvCommMessage(int fd, std::string &msg) {
+    size_t size;
+    if (!recvCommMessageType(fd, &size, sizeof(size))) {
+        return false;
+    }
+    msg.resize(size);
+    return recvCommMessageType(fd, msg.data(), size, true);
```

Cppcheck (2.21.0)

[style] The function 'addErrorTelemetry' is never used.

(unusedFunction)